client/components/Main/HistoryTab.jsx

```diff
@@ -1,23 +1,16 @@
 import React from 'react';
-import {gettext} from '../../utils';
+import PropTypes from 'prop-types';
+import {getItemType} from '../../utils';
+import {EventHistory} from '../Events/';
 
-export const HistoryTab = () => {
-    const items = [
-        {
-            _id: 1,
-            author: 'Mika',
-            created: new Date(),
-        },
-    ];
+export const HistoryTab = ({item}) => (
+    <div>
```

Comment:
If the item type is `Event`, directly return the `EventHistory` component (removing the parent div node, as it is not required)